webpack/scenes/ContentViews/components/NeedsPublishIcon.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { translate as __ } from 'foremanReact/common/I18n';
+import PropTypes from 'prop-types';
+import { Tooltip } from '@patternfly/react-core';
+import { ArrowCircleUpIcon } from '@patternfly/react-icons';
+
+const NeedsPublishIcon = ({ composite }) => (
+  <Tooltip
+    position="auto"
+    enableFlip
+    entryDelay={400}
+    content={composite ? __('Available content view version updates.') :
+      __('Available repositories and/or filters updates.')}
```

Comment:
```suggestion
      __('Publish update available: repositories and/or filters have changed.')}
```
what do you think about something like this instead?

Reply:
Updated. 👍🏼